### PR TITLE
Moving subjects up, show more, nojs

### PR DESCRIPTION
**Pull request by mekarpeles**
Supplants #6250

### CODE REVIEW

**Comment by mekarpeles** [comment]
See http://ol-dev1.us.archive.org:1337/works/OL863808W/Ivanhoe

File: openlibrary/plugins/openlibrary/js/readmore.js [hunk 1]
```diff
@@ -1,4 +1,14 @@
 export function initReadMoreButton() {
+    /*
+      Clamper shows used to show more/less by toggling `hidden`
+      style on parent .clamp tag
+     */
+    $('.clamp').on('click', function(){
+	const up = $(this);
+	if (up.hasClass('clamp')) {
+            up.css({"display": up.css("display") === "-webkit-box" ? "unset" : "-webkit-box"});
+	}
```
Comment (mekarpeles) [inline openlibrary/plugins/openlibrary/js/readmore.js:10]:
```suggestion
        }
```

File: openlibrary/plugins/openlibrary/js/readmore.js [hunk 2]
```diff
@@ -1,4 +1,14 @@
 export function initReadMoreButton() {
+    /*
+      Clamper shows used to show more/less by toggling `hidden`
+      style on parent .clamp tag
+     */
+    $('.clamp').on('click', function(){
+	const up = $(this);
+	if (up.hasClass('clamp')) {
+            up.css({"display": up.css("display") === "-webkit-box" ? "unset" : "-webkit-box"});
```
Comment (mekarpeles) [inline openlibrary/plugins/openlibrary/js/readmore.js:9]:
```suggestion
            up.css({display: up.css('display') === '-webkit-box' ? 'unset' : '-webkit-box'});
```

File: openlibrary/plugins/openlibrary/js/readmore.js [hunk 3]
```diff
@@ -1,4 +1,14 @@
 export function initReadMoreButton() {
+    /*
+      Clamper shows used to show more/less by toggling `hidden`
+      style on parent .clamp tag
+     */
+    $('.clamp').on('click', function(){
+	const up = $(this);
+	if (up.hasClass('clamp')) {
```
Comment (mekarpeles) [inline openlibrary/plugins/openlibrary/js/readmore.js:8]:
```suggestion
        if (up.hasClass('clamp')) {
```

File: openlibrary/plugins/openlibrary/js/readmore.js [hunk 4]
```diff
@@ -1,4 +1,14 @@
 export function initReadMoreButton() {
+    /*
+      Clamper shows used to show more/less by toggling `hidden`
+      style on parent .clamp tag
+     */
+    $('.clamp').on('click', function(){
+	const up = $(this);
```
Comment (mekarpeles) [inline openlibrary/plugins/openlibrary/js/readmore.js:7]:
```suggestion
        const up = $(this);
```

File: openlibrary/macros/SubjectTags.html [hunk 5]
```diff
@@ -1,10 +1,10 @@
-$def with(work,location,tags=[])
+$def with(work, tags=[])
 
 $def render_subjects(label, subjects, track_value, prefix=""):
   $if subjects:
     <div class="section link-box">
-      <h6>$label</h6>
-      <span>
+      <span class="clamp">
+	<h6>$label</h6>
```
Comment (jimchamp) [inline openlibrary/macros/SubjectTags.html:7]:
```suggestion
      <span class="clamp">
      <h6>$label</h6>
```
Replace tab with spaces.

File: openlibrary/templates/type/edition/view.html [hunk 6]
```diff
@@ -235,17 +235,16 @@ <h4 class="year">&mdash;
         </div>
 
     <div class="subjects">
-        <div class="subjects-content restricted-view">
-        $:macros.SubjectTags(work,"overview", ["Subjects"])
-        </div>
-        $:macros.ReadMore("subjects")
+      <div class="subjects-content">
+	$ component_times['SubjectsTags'] = time()
+        $:macros.SubjectTags(work, ["Subjects", "People", "Places", "Times"])
```
Comment (jimchamp) [inline openlibrary/templates/type/edition/view.html:240]:
```suggestion
        $ component_times['SubjectsTags'] = time()
        $:macros.SubjectTags(work, ["Subjects", "People", "Places", "Times"])
```
Replace tab with spaces.

**Review by jimchamp — APPROVED** [review]
Lgtm!  Will merge into book page integration branch.

Some things we may want to add before merging into main:
- Changing the cursor to `pointer` when hovering over the `+`
- Somehow make it more clear that subjects are fully expanded (it's somewhat difficult to tell when all of the subjects span a single line).

**Pull request merged**